src/rp2/locales/es/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,757 @@
+
+msgid "Sum of fiat cost to acquire the balance of crypto based on in-flow transactions"
+msgstr "Suma del costo de fiat para adquirir el saldo de cripto basado en transacciones de entrada"
+
+msgid "Sum of fiat cost to acquire the balance of crypto based on in-flow transactions divided by the crypto balance"
+msgstr "Suma del costo de fiat para adquirir el saldo de cripto basado en transacciones de entrada dividido por el saldo de cripto"
+
+msgid "Summary"
+msgstr "Resumen"
+
+msgid "Summary year"
+msgstr "Año de resumen"
+
+msgid "Table Types"
+msgstr "Tipos de tabla"
+
+msgid "Taxable"
+msgstr "Imponible"
+
+msgid "Taxable Event"
+msgstr "Evento imponible"
+
+msgid "Taxable Event Direction/Type"
+msgstr "Dirección/tipo de evento imponible"
+
+msgid "Taxable Event Fraction %"
+msgstr "Fracción de evento imponible %"
+
+msgid "Taxable Event Fraction Description"
+msgstr "Descripción de fracción de evento imponible"
+
+msgid "Taxable Event Spot Price"
+msgstr "Precio spot de evento imponible"
+
+msgid "Taxable Event Timestamp"
+msgstr "Marca de tiempo de evento imponible"
+
+msgid "Taxable Event {}"
+msgstr "Evento imponible {}"
+
+msgid "Taxable Event {} Amount Fraction"
+msgstr "Cantidad de Fracción del evento imponible {}"
+
+msgid "Taxable Total"
+msgstr "Total imponible"
+
+msgid "This file contains two versions of the same report. One version is grouped by Asset, the other by Asset and Exchange (or wallet)"
+msgstr "Este archivo contiene dos versiones del mismo informe. Una versión se agrupa por activo y la otra por activo e intercambio (o billetera)."
+
+msgid "This report leverages the details of your transactions to give you a snapshot of the value of your unsold holdings"
+msgstr "Este informe utiliza los detalles de sus transacciones para darle una instantánea del valor de sus tenencias no vendidas."
+
+msgid "Time at which the in-transaction occurred"
+msgstr "Hora en que se realizó la transacción interna."
+
+msgid "Time at which the taxable event occurred"
+msgstr "Hora en que ocurrió el evento imponible."
+
+msgid "Time at which the transaction occurred"
+msgstr "Hora en que se realizó la transacción."
+
+msgid "Timestamp"
+msgstr "Marca de tiempo"
+
+msgid "To Date Filter"
+msgstr "Filtro de fecha hasta"
+
+msgid "To Exchange"
+msgstr "Hacia el intercambio"
+
+msgid "To Holder"
+msgstr "Hacia el titular"
+
+msgid "Total"
+msgstr "Total"
+
+msgid "Total Cost Basis %"
+msgstr "Porcentaje del costo base total"
+
+msgid "Transaction"
+msgstr "Transacción"
+
+msgid "Transaction Type"
+msgstr "Tipo de transacción"
+
+msgid "Transactions that added new crypto to accounts (e.g. buy, etc.). Only EARN-typed transactions are taxable events (interest received on crypto)"
+msgstr "Transacciones que agregaron nueva criptomoneda a las cuentas (compra, etc.). Solo las transacciones de tipo EARN son eventos imponibles (intereses recibidos en criptomoneda)."
```

Comment:
Translate EARN?

Reply:
i left EARN without translation, hard to translate with right meaning.